src/LibSDL2.jl

```diff
@@ -6043,219 +6074,219 @@ struct IMG_Animation
 end
 
 function TTF_FontFaceFamilyName(font)
-    ccall((:TTF_FontFaceFamilyName, libsdl2_ttf), Ptr{Cchar}, (Ptr{TTF_Font},), font)
+    ccall((:TTF_FontFaceFamilyName, libsdl2), Ptr{Cchar}, (Ptr{TTF_Font},), font)
 end
 
 function TTF_FontFaceStyleName(font)
-    ccall((:TTF_FontFaceStyleName, libsdl2_ttf), Ptr{Cchar}, (Ptr{TTF_Font},), font)
+    ccall((:TTF_FontFaceStyleName, libsdl2), Ptr{Cchar}, (Ptr{TTF_Font},), font)
 end
 
 function TTF_GlyphIsProvided(font, ch)
-    ccall((:TTF_GlyphIsProvided, libsdl2_ttf), Cint, (Ptr{TTF_Font}, Uint16), font, ch)
+    ccall((:TTF_GlyphIsProvided, libsdl2), Cint, (Ptr{TTF_Font}, Uint16), font, ch)
 end
 
 function TTF_GlyphMetrics(font, ch, minx, maxx, miny, maxy, advance)
-    ccall((:TTF_GlyphMetrics, libsdl2_ttf), Cint, (Ptr{TTF_Font}, Uint16, Ptr{Cint}, Ptr{Cint}, Ptr{Cint}, Ptr{Cint}, Ptr{Cint}), font, ch, minx, maxx, miny, maxy, advance)
+    ccall((:TTF_GlyphMetrics, libsdl2), Cint, (Ptr{TTF_Font}, Uint16, Ptr{Cint}, Ptr{Cint}, Ptr{Cint}, Ptr{Cint}, Ptr{Cint}), font, ch, minx, maxx, miny, maxy, advance)
 end
 
 function TTF_SizeText(font, text, w, h)
-    ccall((:TTF_SizeText, libsdl2_ttf), Cint, (Ptr{TTF_Font}, Ptr{Cchar}, Ptr{Cint}, Ptr{Cint}), font, text, w, h)
+    ccall((:TTF_SizeText, libsdl2), Cint, (Ptr{TTF_Font}, Ptr{Cchar}, Ptr{Cint}, Ptr{Cint}), font, text, w, h)
 end
 
 function TTF_SizeUTF8(font, text, w, h)
-    ccall((:TTF_SizeUTF8, libsdl2_ttf), Cint, (Ptr{TTF_Font}, Ptr{Cchar}, Ptr{Cint}, Ptr{Cint}), font, text, w, h)
+    ccall((:TTF_SizeUTF8, libsdl2), Cint, (Ptr{TTF_Font}, Ptr{Cchar}, Ptr{Cint}, Ptr{Cint}), font, text, w, h)
 end
 
 function TTF_SizeUNICODE(font, text, w, h)
-    ccall((:TTF_SizeUNICODE, libsdl2_ttf), Cint, (Ptr{TTF_Font}, Ptr{Uint16}, Ptr{Cint}, Ptr{Cint}), font, text, w, h)
+    ccall((:TTF_SizeUNICODE, libsdl2), Cint, (Ptr{TTF_Font}, Ptr{Uint16}, Ptr{Cint}, Ptr{Cint}), font, text, w, h)
 end
 
 function TTF_RenderText_Solid(font, text, fg)
-    ccall((:TTF_RenderText_Solid, libsdl2_ttf), Ptr{SDL_Surface}, (Ptr{TTF_Font}, Ptr{Cchar}, SDL_Color), font, text, fg)
+    ccall((:TTF_RenderText_Solid, libsdl2), Ptr{SDL_Surface}, (Ptr{TTF_Font}, Ptr{Cchar}, SDL_Color), font, text, fg)
 end
 
 function TTF_RenderUTF8_Solid(font, text, fg)
-    ccall((:TTF_RenderUTF8_Solid, libsdl2_ttf), Ptr{SDL_Surface}, (Ptr{TTF_Font}, Ptr{Cchar}, SDL_Color), font, text, fg)
+    ccall((:TTF_RenderUTF8_Solid, libsdl2), Ptr{SDL_Surface}, (Ptr{TTF_Font}, Ptr{Cchar}, SDL_Color), font, text, fg)
 end
 
 function TTF_RenderUNICODE_Solid(font, text, fg)
-    ccall((:TTF_RenderUNICODE_Solid, libsdl2_ttf), Ptr{SDL_Surface}, (Ptr{TTF_Font}, Ptr{Uint16}, SDL_Color), font, text, fg)
+    ccall((:TTF_RenderUNICODE_Solid, libsdl2), Ptr{SDL_Surface}, (Ptr{TTF_Font}, Ptr{Uint16}, SDL_Color), font, text, fg)
 end
 
 function TTF_RenderGlyph_Solid(font, ch, fg)
-    ccall((:TTF_RenderGlyph_Solid, libsdl2_ttf), Ptr{SDL_Surface}, (Ptr{TTF_Font}, Uint16, SDL_Color), font, ch, fg)
```

Comment:
@JamesWrigley  I updated the config, and it seems to be overwriting all of these. I'm not sure if I'm doing something wrong here, or if I should be going back and updating these. I figured I'd commit it and show instead of trying to explain.

Reply:
I cannot reproduce this :thinking: If I clone the repo and regenerate the bindings those lines aren't changed. What Clang.jl version are you using? I know `gen/Project.toml` has compat bounds but I'd actually suggest deleting all of them and running `] up`. Generator projects are typically used to generate bindings for the latest version of packages and they'll never be a dependency, so I don't see any point in having compat bounds for them.